spec/index.bs

```diff
@@ -3313,22 +3334,41 @@ dictionary GPUComputePipelineDescriptor : GPUPipelineDescriptorBase {
                 1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
                 1. Create a new [=invalid=] {{GPUComputePipeline}} and return the result.
         </div>
-</dl>
 
-### {{GPUDevice/createReadyComputePipeline()}} ### {#device-createReadyComputePipeline}
+    : <dfn>createReadyComputePipeline(descriptor)</dfn>
+    ::
+        Creates a {{GPUComputePipeline}}. The returned {{Promise}} resolves when the created pipeline
+        is ready to be used without additional delay.
 
-Same as {{GPUDevice/createComputePipeline()}}, but returns its result as a
-promise which doesn't resolve until the pipeline is ready to be used without
-additional delay.
+        If pipeline creation fails, the returned {{Promise}} resolves to an [=invalid=]
+        {{GPUComputePipeline}} object.
 
-If pipeline creation fails, this resolves to an [=invalid=] {{GPUComputePipeline}} object.
+        Note: Use of this method is preferred whenever possible, as it prevents blocking the
+        [=queue timeline=] work on pipeline compilation.
```

Comment:
I think this should be `[=device timeline=]` which refers to the GPU process timeline. The queue timeline is the timeline of work actually being executed on the GPU.

Reply:
I wrote this too; I think it's the queue timeline because the submit() blocks on the pipeline compilation finishing. It depends on whether you define submit() as:

- issues a device timeline task that waits for pipeline compilation then issues a queue timeline task
- issues a queue timeline task that waits for pipeline compilation

but I guess it should be the first one?